### PR TITLE
chore: update core to 0.20.0

### PR DESCRIPTION
**Pull request by dyladan**
This is the minimum updates to take advantage of the latest API. It does not take advantage of new features like startActiveSpan

### CODE REVIEW

File: examples/express/package.json [hunk 1]
```diff
@@ -28,15 +28,15 @@
...
+    "@opentelemetry/instrumentation": "^0.20.0",
     "@opentelemetry/instrumentation-express": "^0.15.0",
-    "@opentelemetry/instrumentation-http": "^0.19.0",
-    "@opentelemetry/node": "^0.19.0",
-    "@opentelemetry/tracing": "^0.19.0",
-    "axios": "^0.19.0",
+    "@opentelemetry/instrumentation-http": "^0.20.0",
+    "@opentelemetry/node": "^0.20.0",
+    "@opentelemetry/tracing": "^0.20.0",
+    "axios": "^0.20.0",
```
Comment (Flarna) [inline examples/express/package.json:39]:
```suggestion
    "axios": "^0.21.1",
```

File: examples/hapi/package.json [hunk 2]
```diff
@@ -32,15 +32,15 @@
...
+    "@opentelemetry/instrumentation": "^0.20.0",
     "@opentelemetry/instrumentation-hapi": "^0.15.0",
-    "@opentelemetry/instrumentation-http": "^0.19.0",
-    "@opentelemetry/node": "^0.19.0",
-    "@opentelemetry/tracing": "^0.19.0",
-    "axios": "^0.19.0"
+    "@opentelemetry/instrumentation-http": "^0.20.0",
+    "@opentelemetry/node": "^0.20.0",
+    "@opentelemetry/tracing": "^0.20.0",
+    "axios": "^0.20.0"
```
Comment (Flarna) [inline examples/hapi/package.json:43]:
```suggestion
    "axios": "^0.21.1"
```

File: examples/koa/package.json [hunk 3]
```diff
@@ -32,15 +32,15 @@
...
+    "@opentelemetry/exporter-zipkin": "^0.20.0",
+    "@opentelemetry/instrumentation": "^0.20.0",
+    "@opentelemetry/instrumentation-http": "^0.20.0",
     "@opentelemetry/instrumentation-koa": "^0.15.0",
-    "@opentelemetry/node": "^0.19.0",
-    "@opentelemetry/tracing": "^0.19.0",
-    "axios": "^0.19.0",
+    "@opentelemetry/node": "^0.20.0",
+    "@opentelemetry/tracing": "^0.20.0",
+    "axios": "^0.20.0",
```
Comment (Flarna) [inline examples/koa/package.json:43]:
```suggestion
    "axios": "^0.21.1",
```

File: examples/network/package.json [hunk 4]
```diff
@@ -32,7 +32,7 @@
     "@opentelemetry/exporter-zipkin": "^0.18.2",
     "@opentelemetry/instrumentation": "^0.18.2",
```
Comment (Flarna) [inline examples/network/package.json:33]: There are some more dependencies we forgot to update last time

File: examples/react-load/preact/package.json [hunk 5]
```diff
@@ -37,11 +37,11 @@
     "sirv-cli": "1.0.3"
   },
   "dependencies": {
-    "@opentelemetry/context-zone": "^0.19.0",
-    "@opentelemetry/exporter-collector": "^0.19.0",
+    "@opentelemetry/context-zone": "^0.20.0",
+    "@opentelemetry/exporter-collector": "^0.20.0",
     "@opentelemetry/plugin-react-load": "^0.15.0",
```
Comment (Flarna) [inline examples/react-load/preact/package.json:42]: almost all examples point to an outdated version of itself. But maybe fix this in a follup

File: examples/redis/package.json [hunk 6]
```diff
@@ -30,15 +30,15 @@
...
+    "@opentelemetry/exporter-zipkin": "^0.20.0",
+    "@opentelemetry/instrumentation": "^0.20.0",
+    "@opentelemetry/instrumentation-http": "^0.20.0",
     "@opentelemetry/instrumentation-redis": "^0.15.0",
-    "@opentelemetry/node": "^0.19.0",
-    "@opentelemetry/tracing": "^0.19.0",
-    "axios": "^0.19.0",
+    "@opentelemetry/node": "^0.20.0",
+    "@opentelemetry/tracing": "^0.20.0",
+    "axios": "^0.20.0",
```
Comment (Flarna) [inline examples/redis/package.json:41]:
```suggestion
    "axios": "^0.21.1",
```

File: plugins/node/opentelemetry-instrumentation-winston/src/winston.ts [hunk 7]
```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright The OpenTelemetry Authors
```
Comment (Flarna) [inline plugins/node/opentelemetry-instrumentation-winston/src/winston.ts:2]: I assume this file was added per accident

File: plugins/node/opentelemetry-instrumentation-bunyan/src/bunyan.ts [hunk 8]
```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright The OpenTelemetry Authors
```
Comment (Flarna) [inline plugins/node/opentelemetry-instrumentation-bunyan/src/bunyan.ts:2]: I assume this file was added per accident

File: examples/koa/package.json (hunk 3 above)
Comment (dyladan) [reply examples/koa/package.json:43]: are we really using a 0.x axios dep?

File: examples/react-load/preact/package.json (hunk 5 above)
Comment (dyladan) [reply examples/react-load/preact/package.json:42]: Will do in follow up good catch

**Comment by dyladan** [comment]
Might be helpful to ignore white space on review since the linter made some changes to indents

File: examples/koa/package.json (hunk 3 above)
Comment (Flarna) [reply examples/koa/package.json]: according to npm 0.21.1 is the latest version.

**Review by Flarna — APPROVED** [review]

**Review by naseemkullah — APPROVED** [review]

File: examples/restify/package.json [hunk 9]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "restify-example",
   "private": true,
-  "version": "0.18.0",
+  "version": "0.15.0",
```
Comment (MSNev) [inline examples/restify/package.json:4]: Looking at the history of this file -- the first version was checked in as 0.18.0.

**Review by weyert — APPROVED** [review]

File: examples/restify/package.json (hunk 9 above)
Comment (dyladan) [reply examples/restify/package.json:4]: Most likely core was 18 at that time and the original contributor was confused what our policy is?

**Comment by lizthegrey** [comment]
Thank you much!

**Comment by dyladan** [comment]
Still waiting on https://github.com/open-telemetry/opentelemetry-js-contrib/pull/520 for the instrumentation to be useable with this sdk and the API it points at but things are looking like it should be done in the next few minutes

**Pull request merged**